### DISCUSSION TITLE
[solved] Why can't I see the score of my css battle account?

### DISCUSSION THREAD

**Discussion by Hacker-C** [cssbattle/forum#185] [Q&A]
Since my score reached 10000，I cant't see my score and it keeps showing "Fetching player data...".
I'm sure that my internet speed is fast.

**Comment by Hacker-C** [comment]
I have solved the problem.

**Reply by co0kie** [reply]
same, how did you fix it?

**Reply by Hacker-C** [reply]
I checked the browser console of the website and I found the GET status of the "getRank" is 502, so there was an error with the server.
As the solution, sometimes I refresh the website and then my score and rank will show, but sometimes not.
So the only thing we can do is to wait for them to optimize the server.

**Reply by co0kie** [reply]
for me, it worked with Incognito mode.

**Comment by chinchang** [comment]
There seems to be some performance issue on the server side where rank is calculated. We are working on it and will be fixed soon.

**Reply by 3rfan** [reply]
welp, 4 months later, I have the same issue!

**Reply by ghost** [reply]
I have same issue.
![fetching-player-data](https://user-images.githubusercontent.com/81631443/156495184-c4fefe2f-60b1-4ada-9c3b-4da6e2da53d4.jpg)

**Comment by AryanKuAg** [comment]
6 March 2022, Still having issue with it.😔
![image](https://user-images.githubusercontent.com/61811630/156919694-3c8ff5ad-587e-4339-9504-f0ae99922ea5.png)

**Reply by umanandt** [reply]
please fix this fetching data issue ...

**Reply by chinchang** [reply]
this is finally fixed!

**Comment by chinchang** [comment]
We made some optimizations on the backend and this issue is now fixed, finally!
Let us know if anyone still faces this issue.

**Reply by AryanKuAg** [reply]
OMG. Its working, A GREAT THANKS TO YOU😍😍😍😍.
![image](https://user-images.githubusercontent.com/61811630/159976639-a2acf5bb-8dd5-4c71-b81d-4f856bbfc4a3.png)
